source/switching_to_live/index.html.md.erb

```diff
@@ -107,7 +107,7 @@ You make a real payment using either:
 
     You must use a real payment card. Use a small amount, because GOV.UK Pay will take the payment from the card you use.
 
-    If you want to see the payment in your bank account, make sure the payment amount is [above the minimum payout](/integrate_with_govuk_pay/#receiving-payments).
+    If you want to see the payment in your bank account, make sure the payment amount will be [above the minimum payout](/integrate_with_govuk_pay/#receiving-payments) after PSP fees are taken. If your PSP is Stripe, we recommend a test payment of at least £2.
```

Comment:
There's passive voice here (and below) but I think making it active would be awkward.

Reply:
How about this?

> If you want to see the payment in your bank account, make sure the payment amount will be above the minimum payout after your PSP has taken their fees.